### ISSUE TITLE
Error with Point mutation identification in v3.6.7

### ISSUE THREAD

**Issue by amridom** [ncbi/amr#21]
Firstly I want to thank you for this oustanding tool.
Since upgrading AMRFinderPlus to v3.6.7, while using point mutation identification feauture, below error appears. I tried installation with Bioconda and binary release methods and both result to the same error. I am running AMRFinderPlus on Windows 10 Subsystem for Linux (Ubuntu 1804). It would be appreciated if you can help me with this issue.

Species : E. coli (Sakai) with NCBI accession number of NC_002695

command and result for binary release method: 
````
./amrfinder -n ./Sakai\ NC_002695.fasta --database ./data/2020-01-22.1/ -O Escherichia --plus --threads 8 --mutation_all ./mutation_result.csv -o ./result.csv
Running ./amrfinder -n './Sakai NC_002695.fasta' --database ./data/2020-01-22.1/ -O Escherichia --plus --threads 8 --mutation_all ./mutation_result.csv -o ./result.csv
Software version: 3.6.7
Database version: 2020-01-22.1
AMRFinder translated nucleotide and mutation search with database ./data/2020-01-22.1/: /mnt/c/AMRFinderPlus/data/2020-01-22.1
Running blastx...
Running blastn...'

'*** ERROR ***
"alignment.cpp", line 629: start_ref_prev < seqChange. start_ref'

'HOSTNAME: ?
SHELL: /bin/bash
PWD: /mnt/c/AMRFinderPlus
Progam name:  amr_report
Command line: ./amr_report -fam ./data/2020-01-22.1//fam.tab -blastx /tmp/H9dJwz.blastx -dna_len /tmp/H9dJwz.len -organism Escherichia -mutation ./data/2020-01-22.1//AMRProt-mutation.tab -mutation_all ./mutation_result.csv -force_cds_report -pseudo -coverage_min 0.5 -suppress_prot /tmp/H9dJwz.suppress_prot -log /tmp/H9dJwz.log
````

command and result for Bioconda installation method:
````
amrfinder -n ./Sakai\ NC_002695.fasta -O Escherichia --plus --threads 8 --mutation_all ./mutation_result2.csv -o ./result2.csv
Running amrfinder -n './Sakai NC_002695.fasta' -O Escherichia --plus --threads 8 --mutation_all ./mutation_result2.csv -o ./result2.csv
Software version: 3.6.7
Database version: 2020-01-22.1
AMRFinder translated nucleotide and mutation search with database /home/amir/miniconda3_20200204/share/amrfinderplus/data/latest: /home/amir/miniconda3_20200204/share/amrfinderplus/data/2020-01-22.1
Running blastx...
Running blastn...

*** ERROR ***
"alignment.cpp", line 629: start_ref_prev < seqChange. start_ref

HOSTNAME: ?
SHELL: /bin/bash
PWD: /mnt/c/AMRFinderPlus
Progam name:  amr_report
Command line: /home/amir/miniconda3_20200204/bin/amr_report -fam /home/amir/miniconda3_20200204/share/amrfinderplus/data/latest/fam.tab -blastx /tmp/qxj0IC.blastx -dna_len /tmp/qxj0IC.len -organism Escherichia -mutation /home/amir/miniconda3_20200204/share/amrfinderplus/data/latest/AMRProt-mutation.tab -mutation_all ./mutation_result2.csv -force_cds_report -pseudo -coverage_min 0.5 -suppress_prot /tmp/qxj0IC.suppress_prot -log /tmp/qxj0IC.log
````

as presented below, when `--mutation_all` option is removed, it works properly:
````
amrfinder -n ./Sakai\ NC_002695.fasta -O Escherichia --plus --threads 8 -o ./result3.csv
Running amrfinder -n './Sakai NC_002695.fasta' -O Escherichia --plus --threads 8 -o ./result3.csv
Software version: 3.6.7
Database version: 2020-01-22.1
AMRFinder translated nucleotide and mutation search with database /home/amir/miniconda3_20200204/share/amrfinderplus/data/latest: /home/amir/miniconda3_20200204/share/amrfinderplus/data/2020-01-22.1
Running blastx...
Running blastn...
AMRFinder took 228 seconds to complete
````

Thanks indeed for your help and support.
Regards,
Amir

**Comment by evolarjun** [comment]
Thanks Amir, I thought I was the only one that used the --mutation-all option :-)

I am able to reproduce the bug and I'll post when we've figured out what's going on.

**Comment by evolarjun** [comment]
This bug was hit only with the point mutation database for Escherichia which is why we had missed it earlier. Our tests no longer miss it, and I also added it to the public tests. I've merged to master and we're going to make a release of version 3.6.10 today. It may take a little while to percolate through bioconda, though I'll do my best to hurry it along. 

Let us know if you're still having issues, and thanks for your help.
